app/models/katello/kt_environment.rb

```
@@ -161,7 +161,7 @@ def deletable?
              " Please change or remove the associated Activation Keys before trying to delete this lifecycle environment.") % self.name)
       end
 
-      return errors.empty?
+      throw :abort unless errors.empty?
```

What's the reason for throwing an error here instead of returning a boolean?